student-library/src/main/java/tds/student/sql/data/OpportunityItem.java

```diff
@@ -58,6 +53,7 @@ public long getItemKey () {
     return _itemKey;
   }
 
+  @JsonProperty("ItemKey")
```

Comment:
@wtritch - in another pull request you asked why this is needed.  When serializing the object to JSON we get this:

```
[
  {
    "Value": "value",
    "Position": 0,
    "Format": "format",
    "ItemKey": 1234,
    "IsRequired": true,
    "Sequence": 1,
    "BankKey": 187,
    "Segment": 1,
    "SegmentID": "segmentId",
    "ItemFile": "itemFile",
    "GroupItemsRequired": 1,
    "GroupID": "groupId",
    "StimulusFile": "stimulusFile",
    "Page": 1,
    "IsSelected": true,
    "IsVisible": true,
    "IsPrintable": false,
    "MarkForReview": false,
    "IsValid": false
  }
]
```

When our configured object mapper tries to read that back into an object nothing maps because the private properties all start with "_".  Syncing the JsonProperty to the getters and setters fixes it.